src/Paystack/Helpers/Router.php

```diff
@@ -96,10 +96,10 @@ private function getAllRoutes($paystackObj)
 
     private function getRouteClass($paystackObj)
     {
-        try {
+        if (isset($paystackObj->custom_routes[$this->route])) {
```

Comment:
please use tests to ascertain the issue and its fix

Reply:
Done @ibrahimlawal-paystack